src/Repository/Repository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ioana
+ * Date: 12/3/2018
+ * Time: 11:30 PM
+ */
+
+namespace BookReviews\Repository;
+
+/**
+ * Interface Repository
+ * @package BookReviews\Repository
+ */
+interface Repository
```

Comment:
All interfaces should have the suffix `Interface` in their name, i.e. `RepositoryInterface`